app/scripts/lib/app-start.js

```diff
@@ -130,9 +132,7 @@ function (
         this.initializeInterTabChannel()
       ])
       .then(_.bind(this.allResourcesReady, this))
-      .then(function () {
-        self._trackWindowOnError();
-      }, function (err) {
+      .then(null, function (err) {
```

Comment:
actually it is here: https://github.com/rkatic/p/blob/4c56fce5c4c953d54ce621fb8b0382baa224e422/p.js#L613 I will check it again